templates/kubedns.go

```diff
@@ -124,6 +133,14 @@ spec:
                     operator: In
                     values: ["kube-dns"]
               topologyKey: kubernetes.io/hostname
+		nodeAffinity:
```

Comment:
Indentation is broken here ?

Reply:
@moelsayed , em, changed `tab` to `space`.